modules/administration-guide/partials/assembly_configuring-prod-short-with-custom-resouces-to-use-a-backup-server.adoc

```diff
@@ -0,0 +1,18 @@
+:parent-context-of-configuring-prod-short-with-custom-resouces-to-use-a-backup-server: {context}
+
```

Comment:
- I'm confused here. Is `prod-short` the name of the product or should it be a variable, that is `{prod-short}'?
- The filename is misspelled: `resouces` should be `resources`.

Reply:
@bburt-rh, apologies. This file is a leftover that should have been deleted.